docs/pages/experimental/agents.mdx

```diff
@@ -0,0 +1,42 @@
+import { Callout } from "nextra/components"
+
+<Callout type="warning">
+This part of the documentation is for experimental features. The APIs and functionalities are subject to frequent change.
+</Callout>
+
+<Callout type="warning">
+The Agent API implemented here conflicts with stable Agent API in Sotopia.
+</Callout>
+
+Agent is a concept in Sotopia to represent decision-making entities that can interact with each other in a social environment. Agents can be human participants, AI models, or other entities.
+No matter which type of agent, they have the same interface to interact with the environment:
+the input and output are of derived types of `aact.messages.DataModel`.
+
+### Creating your own agents
+To create your own agents, you need to subclass the `BaseAgent` class
+and implement the asynchronous `aact` method.
+The `aact` method takes an `Observation` object as input and returns an `AgentAction` object as output. Here is an example of a simple agent that always says "Hello, world!":
+
+```python
+from aact import NodeFactory
+from aact.messages import Text
+from sotopia.experimental import BaseAgent
+
+@NodeFactory.register("simple_echo_agent")  # Register the agent so that it can be used in the dataflow
+class SimpleEchoAgent(BaseAgent[Text, Text]):
+    def __init__(self, input_channel: str, output_channel: str, redis_url: str) -> None:
```

Comment:
Could you explain a bit more about channel here?

Reply:
Just updated the doc